lib/screens/settings/qr.dart

```diff
@@ -32,20 +32,27 @@ class QRDisplay extends StatelessWidget {
                     borderRadius: BorderRadius.circular(querySize.width * 0.06),
                     color: Colors.white,
                   ),
-                  child: QrImageView(
-                    data: inviteLink,
-                    eyeStyle: const QrEyeStyle(
-                      eyeShape: QrEyeShape.square,
-                      color: Colors.black,
-                    ),
-                    dataModuleStyle: const QrDataModuleStyle(
-                      dataModuleShape: QrDataModuleShape.square,
-                      color: Colors.black,
-                    ),
-                    embeddedImage: userChannel?.profilePicture,
-                    embeddedImageStyle: const QrEmbeddedImageStyle(
-                      size: Size(60, 60),
-                    ),
+                  child: Stack(
+                    alignment: Alignment
+                        .center, // This centers the CircleAvatar in the middle of the Stack
+                    children: [
+                      QrImageView(
+                        data: inviteLink,
+                        eyeStyle: const QrEyeStyle(
+                          eyeShape: QrEyeShape.square,
+                          color: Colors.black,
+                        ),
+                        dataModuleStyle: const QrDataModuleStyle(
+                          dataModuleShape: QrDataModuleShape.square,
+                          color: Colors.black,
+                        ),
+                        // Remove the embeddedImage property
+                      ),
+                      CircleAvatar(
+                        radius: 35,
```

Comment:
Is this a fine amount? I picked 35 as usually the images we use are 70x70
![Screen Shot 2023-12-27 at 2 48 15 AM](https://github.com/muxable/rtchat/assets/100245448/f2e9d155-0e9d-4e9e-8094-976728926a0c)